<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastros">
    <property name="title">
     <string>Cadastros</string>
    </property>
    <addaction name="actionProdutos"/>
    <addaction name="actionClientes"/>
    <addaction name="actionUser"/>
   </widget>
   <widget class="QMenu" name="menuOperacional">
    <property name="title">
     <string>Operacional</string>
    </property>
    <addaction name="actionCaixa"/>
    <addaction name="actionEstoque"/>
   </widget>
   <widget class="QMenu" name="menuRelatorios">
    <property name="title">
     <string>Relatórios</string>
    </property>
   </widget>
   <addaction name="menuCadastros"/>
   <addaction name="menuOperacional"/>
   <addaction name="menuRelatorios"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProdutos">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionCaixa">
   <property name="text">
    <string>Caixa</string>
   </property>
  </action>
  <action name="actionEstoque">
   <property name="text">
    <string>Estoque</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>Usuário</string>
   </property>
  </action>
  <action name="actionCalculando_Markup">
   <property name="text">
    <string>Calculando Markup</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
